algorithms/C/tree/MinElementOfTree.c

```diff
@@ -0,0 +1,106 @@
+// Wap to find the minimum(using recursion) element in a binary search tree.

```

Comment:
Please change Wap to Way and write using recursion as written below

// Way to find the minimum element in a binary search tree (using recursion)